app/models/user.rb

```diff
@@ -303,6 +346,11 @@ class User < ApplicationRecord
   after_save :subscribe_to_mailchimp_newsletter
 
   after_create_commit :send_welcome_notification
+
+  # NOTE: @msarit `sync_users_settings_table` blows up when trying to sync the
+  # User.profile fields, probably because the user's profile hasn't been created yet
```

Comment:
We'll most likely tackle the profile fields in the ProfileMigration concern above

Reply:
> probably because the user's profile hasn't been created yet

Profiles should get created when the user gets created, so hopefully that's not it.

```ruby
# All new users should automatically have a profile
after_create_commit -> { Profile.create(user: self) }, unless: :_skip_creating_profile
```